clash_lib/src/proxy/converters/tuic.rs

```diff
@@ -37,7 +42,9 @@ impl TryFrom<&OutboundTuic> for AnyOutboundHandler {
                 .unwrap_or_default(),
             heartbeat_interval: Duration::from_millis(s.heartbeat_interval.unwrap_or(3000)),
             reduce_rtt: s.reduce_rtt.unwrap_or(false) || s.fast_open.unwrap_or(false),
-            request_timeout: Duration::from_millis(s.request_timeout.unwrap_or(8000)),
+            request_timeout: Duration::from_millis(s.request_timeout.unwrap_or(4000)),
+            idle_timeout: VarInt::from_u64(s.request_timeout.unwrap_or(4000))
+                .unwrap_or(VarInt::MAX),
```

Comment:
could you use Duration here too?

Also if it failed to pass we should use the same default value 4000 rather than MAX?